package.json

```diff
@@ -2382,6 +2382,15 @@
                     "light": "images/light/icon-checkout.svg"
                 }
             },
+            {
+                "command": "gitlens.views.pruneRemote",
+                "title": "Prune Remote",
```

Comment:
Since this command will only show up on a remote, let's just call it `Prune`

Reply:
should the methods in `viewCommands.ts`, `gitService.ts`, and `git.ts` also just be called `prune`? Or should I leave them as is?

Reply:
Good as is. Thanks!